src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -1038,6 +1038,7 @@ function MOI.delete(model::Optimizer, indices::Vector{<:MOI.VariableIndex})
     # We throw away name_to_constraint_index so we will rebuild VariableIndex
     # constraint names without v.
     model.name_to_constraint_index = nothing
+    # _require_update(model, model_change = true)
     _update_if_necessary(model, force = true)
```

Comment:
This is the only update remaining for variable deletion as is needed for the `test_Buffered_deletion_test` case (linked earlier)